source/_templates/layout.html

```diff
@@ -2,13 +2,13 @@
 
 {%- block htmltitle %}
 <!-- Jquery already required by bootstrap, and two uses of $(document), current document site throwing js errors without this -->
-  <script src="/_static/jquery.min.js"></script>
-  <script src="/_static/lucide/lucide.min.js"></script>
+  <script src="https://openfisca.org/doc/_static/jquery.min.js"></script>
+  <script src="https://openfisca.org/doc/_static/lucide/lucide.min.js"></script>
 
   {% if trainingtemplate %}
-  <link rel="stylesheet" href="/_static/reveal/reveal.css">
-  <link rel="stylesheet" href="/_static/css/components/presentation.css">
-  <link rel="stylesheet" href="/_static/reveal/theme/openfisca.css" id="theme">
+  <link rel="stylesheet" href="../_static/reveal/reveal.css">
+  <link rel="stylesheet" href="../_static/css/components/presentation.css">
+  <link rel="stylesheet" href="../_static/reveal/theme/openfisca.css" id="theme">
```

Comment:
```suggestion
  <script src="{{ pathto('_static/jquery.min.js', 1) }}"></script>
  <script src="{{ pathto('_static/lucide/lucide.min.js', 1) }}"></script>

  {% if trainingtemplate %}
  <link rel="stylesheet" href="{{ pathto('_static/reveal/reveal.css', 1) }}">
  <link rel="stylesheet" href="{{ pathto('_static/css/components/presentation.css', 1) }}">
  <link rel="stylesheet" href="{{ pathto('_static/reveal/theme/openfisca.css', 1) }}" id="theme">
```

Reply:
Thanks for the suggestion, I could not for some reason commit your suggestion from github so have tested this locally and applied your changes

Reply:
I missed this during my review, but I think that image inclusions in markdown pages should also use `pathto`.

Reply:
I attempted this but it was not parsed in the final html and remained unprocessed. I have no reason to believe the relative url's will be a problem in this situation.